### PR TITLE
Add suffixes to typos that start with "f"

### PR DESCRIPTION
**Pull request by int-y1**
I moved `forewarded` from dictionary_rare to dictionary.

### CODE REVIEW

**Comment by DimitriPapadopoulos** [comment]
> I moved `forewarded` from dictionary_rare to dictionary.

[According to OED, `foreward`](https://www.oed.com/search/dictionary/?q=foreward) remains an acceptable verb. On the other hand, it is not part of the [`en_GB-large` _aspell_ dictionary](http://app.aspell.net/lookup?dict=en_GB-large;words=foreward).

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -24448,13 +24496,19 @@ finializing->finalizing
...
+finisches->finishes
+finisching->finishing
 finised->finished
+finishd->finished
 finishe->finished, finish,
 finishied->finished
 finishs->finishes
+finising->finishing
 finitel->finite
 finness->finesse
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:24508]: `finness` coule be a misspelling of `fineness`, perhaps even `fitness`.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24834,9 +24911,14 @@ fonctionnalities->functionalities
...
 fonctionnaly->functionally, functionality,
 fonctions->functions
+fondamental->fundamental
+fondamentally->fundamentally
+fondamentals->fundamentals
 fonetic->phonetic
 fontain->fountain, contain,
 fontains->fountains, contains,
+fontend->frontend, contend,
+fontends->frontends, contends,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:24921]: Perhaps add `fontended` and `fontending`, unless they are too rare.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -24916,16 +25010,28 @@ formattind->formatting
...
 formes->forms, formed,
 formidible->formidable
+formmat->format
+formmats->formats
 formmatted->formatted
+formmatter->formatter
+formmatters->formatters
+formmatting->formatting
 formost->foremost
 formt->format
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:25029]: Is `formt` never a typo of `form`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -25050,14 +25169,21 @@ framwework->framework
...
+franework->framework
+franeworks->frameworks
 frankin->franklin
 Fransiscan->Franciscan
 Fransiscans->Franciscans
 franzise->franchise
 fration->fraction, ration,
 frational->fractional, rational,
 frations->fractions, rations,
+freame->frame
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:25183]: Is `freame` ever a typo of `cream`? In any case, what about `freamed->creamed`?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:24921]: These are too rare.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:25029]: `formt->form` exists, but is rare. I will add it.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:24508]: `finness->fineness` exists. `finness->fitness` exists but is rarer. I'll add both.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:25183]: I couldn't find both of these.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**